src/components/Sum/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    <>
+      <span>Sum of </span>
+      {a}
+      <span> and </span>
+      {b}
+      <span> is </span>
+      {a + b}
```

Comment:
Consider using a template string instead of spans. You can use a span wrapper to add some styling for the text, but there is no additional styling in this task.